<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>908</width>
    <height>696</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="Cauldron" name="widget_cauldron_start" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="frame_pipeflow">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>160</y>
      <width>90</width>
      <height>530</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Box</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="Cauldron" name="widget_cauldron_finish" native="true">
    <property name="geometry">
     <rect>
      <x>750</x>
      <y>540</y>
      <width>150</width>
      <height>150</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_start">
    <property name="geometry">
     <rect>
      <x>384</x>
      <y>323</y>
      <width>140</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>0</y>
      <width>391</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>24</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Игра &quot;Собери трубопровод&quot;</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_level">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>90</y>
      <width>211</width>
      <height>221</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="title">
     <string>Уровень сложности</string>
    </property>
    <widget class="QRadioButton" name="radioButton_easy">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Простой</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_normal">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Средний</string>
     </property>
    </widget>
    <widget class="QRadioButton" name="radioButton_hard">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>150</y>
       <width>141</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Сложный</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_time_txt">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>50</y>
      <width>301</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="mouseTracking">
     <bool>false</bool>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::NoContextMenu</enum>
    </property>
    <property name="text">
     <string>Время до следующего течения воды: </string>
    </property>
    <property name="textFormat">
     <enum>Qt::AutoText</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_time">
    <property name="geometry">
     <rect>
      <x>560</x>
      <y>50</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Times New Roman</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Cauldron</class>
   <extends>QWidget</extends>
   <header location="global">cauldron.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
